{{cookiecutter.project_slug}}/clients/vue3/src/App.vue

```diff
@@ -14,8 +14,10 @@
         |
         <router-link to="/signup" class="router">Signup</router-link>
       </template>
-    </div>
-    <router-view />
+    </nav>
+    <main>
+      <router-view />
+    </main>
```

Comment:
**Info:** Added a `<main>` tag wrapper for the router-view in order to comply with the main body content landmark requirements as defined by WCAG.

Screen shot of issue from Axe Dev tools in Chrome attached below: